Dockerfile

```diff
@@ -1,23 +1,33 @@
-FROM alpine:3.7
+FROM alpine:latest
 LABEL maintainer="Sebastien Lucas <sebastien@slucas.fr>"
 LABEL Description="Home Assistant"
 
 ARG TIMEZONE=Europe/Paris
 ARG UID=1000
 ARG GUID=1000
+ARG MAKEFLAGS=-j4
```

Comment:
Quick question : does this do anything ? cryptography takes a long time to build and it only use one core of my 4 pine64 cores.